chainerx_cc/chainerx/python/array.cc

```diff
@@ -284,6 +284,18 @@ void InitChainerxArray(pybind11::module& m) {
           [](const ArrayBodyPtr& self, int8_t axis1, int8_t axis2) { return MoveArrayBody(Array{self}.Swapaxes(axis1, axis2)); },
           "axis1"_a,
           "axis2"_a);
+    c.def("repeat",
+          [](const ArrayBodyPtr& self, int64_t repeats, nonstd::optional<int8_t> axis) {
+              return MoveArrayBody(Repeat(Array{self}, repeats, axis));
+          },
+          py::arg("repeats"),
+          py::arg("axis") = nullptr);
+    c.def("repeat",
+          [](const ArrayBodyPtr& self, const std::vector<int64_t>& repeats, nonstd::optional<int8_t> axis) {
+              return MoveArrayBody(Repeat(Array{self}, repeats, axis));
+          },
+          py::arg("repeats"),
+          py::arg("axis") = nullptr);
```

Comment:
This I can also change afterwards but we've adapted the less cluttery `_a` literal instead of `py::arg` https://github.com/chainer/chainer/blob/master/chainerx_cc/chainerx/python/routines.cc#L154-L156.

Reply:
fixed